js/jquery.ntm.js

```diff
@@ -14,7 +14,8 @@
         spoilerButtonClickMaxX: 20,
         spoilerButtonClickMinY: 8,
         spoilerButtonClickMaxY: 24,
-        slideEffect: true
+        slideEffect: true,
+        postRender : function () { $.noop }

```

Comment:
Yes. I changed the code as per your comment and tested. I will update the change now. Thanks